$MirOS: ports/devel/cvs/patches/patch-man_cvs_5,v 1.1 2010/09/16 00:16:28 tg Exp $
--- man/cvs.5.orig	Thu Feb 12 21:07:10 2004
+++ man/cvs.5	Sat Sep 18 23:48:23 2010
@@ -1,3 +1,4 @@
+.\" $MirOS$
 .TH cvs 5 "12 February 1992"
 .\" Full space in nroff; half space in troff
 .de SP
@@ -238,9 +239,9 @@ The arguments passed to the command are,
 .B add
 for `tag',
 .B mov
-for `tag -F', and
+for `tag \-F', and
 .B del
-for `tag -d`),
+for `tag \-d`),
 .I repository ,
 and any remaining are pairs of
 .B "filename revision" .
